packages/httpsnippet-client-api/test/__datasets__/full-many-query-params/openapi.json

```diff
@@ -36,6 +36,20 @@
               "type": "string"
             },
             "required": true
+          },
+          {
```

Comment:
Added these because this `full-many-query-params` test is testing some cookie parameters but because they weren't defined here as a cookie parameter they were instead getting sent as query parameters.